timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -288,75 +241,38 @@ def supports_past_covariates(self) -> bool:
-
-    def _init_params_aux(self):
-        """
-        Initializes auxiliary hyperparameters.
-        These parameters are generally not model specific and can have a wide variety of effects.
-        For documentation on some of the available options and their defaults, refer to `self._get_default_auxiliary_params`.
-        """
-        hyperparameters_aux = self._user_params_aux or {}
-        default_aux_params = dict(
-            # ratio of given time_limit to use during fit(). If time_limit == 10 and max_time_limit_ratio=0.3,
-            # time_limit would be changed to 3.
-            max_time_limit_ratio=self.default_max_time_limit_ratio,
-            # max time_limit value during fit(). If the provided time_limit is greater than this value, it will be
-            # replaced by max_time_limit. Occurs after max_time_limit_ratio is applied.
-            max_time_limit=None,
-        )
-        self.params_aux = {**default_aux_params, **hyperparameters_aux}
-
     def _initialize_transforms_and_regressor(self) -> None:
-        self.target_scaler = get_target_scaler(self._get_model_params().get("target_scaler"), target=self.target)
+        self.target_scaler = get_target_scaler(self.get_hyperparameters().get("target_scaler"), target=self.target)
         self.covariate_scaler = get_covariate_scaler(
-            self._get_model_params().get("covariate_scaler"),
+            self.get_hyperparameters().get("covariate_scaler"),
             covariate_metadata=self.metadata,
             use_static_features=self.supports_static_features,
             use_known_covariates=self.supports_known_covariates,
             use_past_covariates=self.supports_past_covariates,
         )
         self.covariate_regressor = get_covariate_regressor(
-            self._get_model_params().get("covariate_regressor"),
+            self.get_hyperparameters().get("covariate_regressor"),
             target=self.target,
             covariate_metadata=self.metadata,
         )
 
-    def _get_model_params(self) -> dict:
-        return self.params.copy()
+    def _get_default_hyperparameters(self) -> dict:
+        return {}
 
-    def get_params(self) -> dict:
-        hyperparameters = self._user_params.copy()
-        if self._user_params_aux:
-            hyperparameters[AG_ARGS_FIT] = self._user_params_aux.copy()
-
-        return dict(
-            path=self.path_root,
-            name=self.name,
-            eval_metric=self.eval_metric,
-            hyperparameters=hyperparameters,
-            freq=self.freq,
-            prediction_length=self.prediction_length,
-            quantile_levels=self.quantile_levels,
-            metadata=self.metadata,
-            target=self.target,
-        )
+    def get_hyperparameters(self) -> dict:
+        """Get hyperparameters provided by the user for passing into the
+        "inner model" that AutoGluon wraps.
+        """
+        return {**self._get_default_hyperparameters(), **self._hyperparameters}
```

Comment:
Let's return a copy to be consistent with the behavior of `_get_model_params`

Reply:
I don't get it. isn't this a shallow copy in the same sense?

```
a = {"key": 5}
b = a
assert a is b

c = {"key": 6}
d = {**c}
assert d is not c
```

Reply:
My bad, `d = {**c}` is indeed equivalent to a shallow copy.